schemas/logipedia.json

```diff
@@ -0,0 +1,121 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "title": "Logipedia object specification",
+    "description": "A single file describing a logipedia object, contains a
+general section and a section dedicated to export systems.",
+    "type": "object",
+    "definitions": {
+        "taxon": {
+            "type": "string",
+            "pattern": "^(axiom|theorem|definition|constant)$"
+        },
+        "logic": {
+            "type": "string",
+            "pattern": "^(coq|matita|lean|pvs|dedukti)$"
```

Comment:
This pattern could evolve, depends on the version of the system. Is it interesting to specify it?

Reply:
we can set it loose indeed